python/paddle/fluid/tests/book/test_word2vec.py

```diff
@@ -220,9 +220,7 @@ def to_infer_tensor(lod_tensor):
         np_data = np.array(results[0])
         infer_out = infer_outputs[0].data.float_data()
         for a, b in zip(np_data[0], infer_out):
-            g_a = float("{:.6g}".format(a))
-            g_b = float("{:.6g}".format(b))
-            assert g_a == g_b
+            assert np.isclose(g_a, g_b)
```

Comment:
isclose, set a retio 1e-4 ?

Reply:
The default relative tolerance parameter is 1e-5, this is the float's accuracy.

Reply:
add some debug information if this assertion fail

such as 

```python
assert np.isclose(...), "g_a: %f, g_b: f" % (g_a, g_b)
```